boards/qemu/cortex_a9/Kconfig.defconfig

```diff
@@ -29,6 +29,9 @@ if NETWORKING
 config NET_L2_ETHERNET
 	default y
 
+config MDIO
```

Comment:
maybe add ``select MDIO if  DT_HAS_XLNX_GEM_MDIO_ENABLED`` to ``menuconfig ETH_XLNX_GEM`` in drivers/ethernet/Kconfig.xlnx_gem instead. could also be a imply instead of a select

Reply:
done